src/main/java/pages7daySuperMarket/AdminUsersPage.java

```diff
@@ -1,183 +1,175 @@
+
+	@FindBy(xpath = "//input[@id='password']")
+	public WebElement passwordInputField;
+
+	@FindBy(xpath = "//select[@id='user_type']")
+	public WebElement userTypeInputField;
+
+	@FindBy(xpath = "//button[@name='Create']")
+	public WebElement saveButton;
+
+	@FindBy(xpath = "//div[@class='card']/div/table/tbody")
+	public WebElement userTable;
+
+	@FindBy(xpath = "//div[@class='card']/div/table/tbody/tr")
+	public List<WebElement> userRows;
+
+	@FindBy(xpath = "//div[@class='card']/div/table/tbody/tr[1]/td[1]")
+	public WebElement firstUserInTable;
+
+	@FindBy(xpath = "//a[@class='btn btn-rounded btn-primary']")
+	public WebElement searchButton;
+
+	@FindBy(xpath = ("//input[@id='un']"))
+	public WebElement searchUserName;
+
+	@FindBy(xpath = ("//select[@id='ut']"))
+	public WebElement searchUserType;
+
+	@FindBy(xpath = ("//button[@name='Search']"))
+	public WebElement submitSearch;
+
+	@FindBy(xpath = ("//a[@class='btn btn-rounded btn-warning']"))
+	public WebElement resetButton;
+
+	@FindBy(xpath = "//h4[text()='Search Admin Users']")
 	public WebElement searchAdminUsersTitle;
-	
-	@FindBy(xpath="//td[text()='Fathima123']/parent::tr/td/a[@class='btn btn-sm btn btn-danger btncss']")	
-	public WebElement deleteIconForFathima123;
-	
-	@FindBy(xpath="//input[@id='username']")
-	public WebElement UsernameInputFieldForFathima123;
-	
-	@FindBy(xpath="//input[@id='password']")
-	public WebElement PasswordInputFieldForFathima123;
-	
-	@FindBy(xpath="//select[@id='user_type']")
-	public WebElement UserTypeInputFieldForFathima123 ;
-	
-	@FindBy(xpath="//button[@name='Create']")
-	public WebElement SaveButtonForFathima123;
-	@FindBy(xpath="//div[@class='alert alert-success alert-dismissible']")
-	public WebElement alertPopUpAfterDeletionOfFathima123;
-	public String validateAlertPopupAfterDeleteUser()
-	{
-	return	alertPopUpAfterDeletionOfFathima123.getText();
-	}
-	
-	public void enterUsernameForFathima123()
-	{
-		UsernameInputFieldForFathima123.sendKeys("Fathima123");
-	}
-	public void clickOnSaveButtonForFathima123()
-	{
-		SaveButtonForFathima123.click();
-	}
-	public void enterUserTypeForFathima123()
-	{
-		UserTypeInputFieldForFathima123.sendKeys("Admin");	
-	}
-	public void enterPasswordForFathima123()
-	{
-		PasswordInputFieldForFathima123.sendKeys("Password");
-	}
-	
-
-	public void clickOnDeleteIconForFathima123()
-	{
-		deleteIconForFathima123.click();
-	}
-	
-	
-	
-	
-	public boolean isTitleSearchAdminUsersDisplayed()
-	{
+
+	@FindBy(xpath = "//td[text()='Fathima123']/parent::tr/td/a[@class='btn btn-sm btn btn-danger btncss']")
+	public WebElement deleteIconForAUser;
+
+	@FindBy(xpath = "//div[@class='alert alert-success alert-dismissible']")
+	public WebElement alertPopUpAfterDeletingAUser;
+
+	@FindBy(xpath = "//a[@data-toggle='dropdown']")
+	public WebElement adminImageAndTextButton;
+
+	@FindBy(xpath = "//a[@class='dropdown-item'][2]")
+	public WebElement LogoutButton;
+
+	public void doLogout() {
+		adminImageAndTextButton.click();
```

Comment:
its a good practise to keep 2 space indented for immediate lines;
you can use code formatting features from eclipse for that